RELEASENOTES.md

```diff
@@ -5,3 +5,5 @@
 - In v0.25 we started publishing `ado2gh` as an extension to the `gh` CLI. However, we didn't update `ado2gh generate-script` to use the new syntax in the generated migration script. Now it will, and you will need the `gh ado2gh` extension installed in order to run the generated migration script.
 - Added `--ghes-api-url` as an optional arg to the `grant-migrator-role` and `revoke-migrator-role` commands for both `ado2gh` and `gei`.
 - Added AWS support for archive uploads to `gei` using the `--aws-bucket-name`, `--aws-access-key` and `--aws-secret-key` arguments.
+- Added `--bbs-shared-home` option to `migrate-repo` and `generate-script` commands for `bbs2gh`. This provides support for BBS servers with a non default directory path.
+- Added support of running the CLI on the Bitbucket server by locating the export archive file and skipping the archive downloading step.
```

Comment:
Let's not mention bbs migrations in the release notes until we go private beta. I removed previous mentions before cutting the release today.